go/teams/chatmessaging.go

```diff
@@ -58,12 +58,12 @@ func SendChatInviteWelcomeMessage(ctx context.Context, g *libkb.GlobalContext, t
 		return false
 	}
 
-	inviterName, err := g.GetUPAKLoader().LookupUsername(ctx, inviter)
+	inviterName, err := g.GetUPAKLoader().LookupUsernameUPAKLite(ctx, inviter)
```

Comment:
how does this not use upak lite in prod?

Reply:
You're right, my bad. I think in addition checking the run mode, the code should only load UPAKLites if a specific feature flag is set that is only set during some tests. Does that sound ok (in addition to fixing these)?